template.distillery/PROJECT_NAME_handlers.eliomi

```diff
@@ -0,0 +1,59 @@
+(* This file was generated by Ocsigen-start.
+   Feel free to use it, modify it, and redistribute it as you wish. *)
+
```

Comment:
Is this a superset of the `Os_handlers` signature? If yes, it would be much more maintainable if we had an `include module type of Os_handlers`.